core/src/main/java/com/scalar/db/util/groupcommit/ThrowableRunnable.java

```diff
@@ -0,0 +1,6 @@
+package com.scalar.db.util.groupcommit;
+
+@FunctionalInterface
+interface ThrowableRunnable {
```

Comment:
We already have `ThrowableRunnable`:
https://github.com/scalar-labs/scalardb/blob/53af316e1badb9460652ef7fcbb54612756edd3b/core/src/main/java/com/scalar/db/util/ThrowableRunnable.java

Reply:
Fixed it in https://github.com/scalar-labs/scalardb/pull/1607/commits/e8faee28e7f1a7f7cdfb2b2d1d36cc8673e34e36